content/source/docs/cloud/sentinel/sentinel-tf-012.html.md

```diff
@@ -17,9 +17,20 @@ modification.
 
 However, due to fundamental changes introduced in Terraform 0.12, some API
 changes were required. While `tfconfig` was most impacted, there have been
-notable changes to both `tfplan` and `tfstate` as well.
+notable changes to both `tfplan` and `tfstate` as well. These are described
+below. Your policies will need to be adjusted if they are affected by
+these changes.
 
-Your policies will need to be adjusted if they are affected by these changes.
+One change that affects the `tfconfig`, `tfplan`, and `tfstate` imports is
+that numeric attributes of resources are now treated as floats in Terraform
+0.12. This could require some modifications if your current policies treat
+these attributes as strings. In particular, you might have to modify existence
+checks and comparisons done against numeric attributes.
+
+There are no explicit changes in the `tfrun` import for Terrraform 0.12, but
+the [`cost_estimate namespace`](./import/tfrun.html#namespace-cost_estimate)
```

Comment:
Should it be:

- `cost_estimate namespace`, or
- `cost_estimate` namespace

I'm low confidence on what the right answer is, just jumped out at me and thought I'd flag it for consideration. Happy to defer judgement to the experts.

Reply:
I believe `cost_estimate namespace` was already used.  And the doc definitely did already have `workspace namespace`.  So, what I did is consistent.

Reply:
I did take a look at this and I think that the issue that Glenn was mentioning here is correct in that the formatting should be "`cost_estimate` namespace". The other one reads "resource namespace" without pre-formatting so it's not _exactly_ the same thing.
 
I fixed this though so it doesn't have to be anymore, this is good to merge now without any more changes.